resolver/query_logging_resolver.go

```diff
@@ -85,6 +90,24 @@ func NewQueryLoggingResolver(cfg config.QueryLogConfig) ChainedResolver {
 	return &resolver
 }
 
+func resolveQueryLogFields(cfg config.QueryLogConfig) []config.QueryLogField {
+	var fields []config.QueryLogField
+
+	if len(cfg.Fields) == 0 {
+		// no fields defined, use all fields as fallback
+		for _, v := range config.QueryLogFieldNames() {
+			qlt, err := config.ParseQueryLogField(v)
+			util.LogOnError("unknown query log field", err)
```

Comment:
Would be nice to be explicit IMO:
```suggestion
			util.LogOnError("ignoring unknown query log field", err)
```